Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1459,6 +1459,49 @@ namespace Babylon
             XRFrame* m_frame{};
         };
 
+        class XRHand : public Napi::ObjectWrap<XRHand>
```

Comment:
Can we move `XRHand` into another file? This one's already getting kind of long.

Reply:
It is, but all the other XR* classes are defined in here as well so I would probably move them all out at that point...

Reply:
I have an issue tracking this here: https://github.com/BabylonJS/BabylonNative/issues/343

If this can be moved into another file we should go ahead and do it to set precedent. I don't know when anyone will get to the above issue.

Reply:
I might have made it a little harder to do this by placing HAND_JOINT_NAMES in an anonymous namespace in NativeXR, but I'll give it a shot